src/script/booking.js

```diff
@@ -16,17 +18,33 @@ function addCancelOption() {
       challengeNumber = "";
       times = "";
       date = "";
+      minPart = "";
+      maxPart = "";
       body.removeChild(close);
       body.style.overflow = "auto";
       return;
     }
   });
 }
 
+function errorInfo(error) {
+  const close = document.querySelector(".booking");
+  const template = `<div class="error"><p>${error}</p></div>`;
+  close.insertAdjacentHTML("afterbegin", template);
+  setTimeout(()=> {
+    const errorDiv = document.querySelector(".error");
+    if(errorDiv) {
+      close.removeChild(errorDiv);
+    }
+  }, 6000);
+}
+
 // Book a room (First Step)
-function book(title, challenge) {
+function book(title, challenge, Min, Max) {
```

Comment:
The naming convention here, with using uppercase `Min` and `Max` (instead of `min` and `max`), is uncommon and not consistent with the code in general. I would recommend against it, as it conveys that there is something special about these parameters when there really isn't.